apps/authentication/routes.py

```diff
@@ -16,13 +16,15 @@
 from apps.config import app_config
 from apps.audit_mixin import get_remote_addr, utcnow
 from apps.authentication import blueprint
-from apps.authentication.forms import LoginForm, CreateAccountForm, ConfirmAccountForm
+from apps.authentication.forms import LoginForm, CreateAccountForm, ConfirmAccountForm, ResetPasswordForm, ResetPasswordConfirmForm
 from apps.authentication.models import Users, Groups, LoginLogging
 from flask_mail import Message
 import uuid
 from datetime import timedelta
+from sqlalchemy import or_
+import secrets
 
-from apps.authentication.util import verify_pass
+from apps.authentication.util import verify_pass, hash_pass
```

Comment:
you dont need to import the  `hash_pass`, this function is encapsulated into `user.set_password()`